### PR TITLE
Upgrade dependencies: 09 2024

### PR DESCRIPTION
**Pull request by KCeh**
## Summary

Just bump in dependencies (including AGP and Gradle wrapper) and compile and target SDK bump to 35

## Changes

- Dependencies updated
- AGP and Gradlew wrapper update
- `compileSdk` and `targetSdk` set to 35

### Type

- [ ] **Feature**: This pull request introduces a new feature.
- [ ] **Bug fix**: This pull request fixes a bug.
- [ ] **Refactor**: This pull request refactors existing code.
- [ ] **Documentation**: This pull request updates documentation.
- [x] **Other**: This pull request makes other changes.

#### Additional information

- [ ] This pull request introduces a **breaking change**.

## Checklist

- [x] I have performed a self-review of my own code.
- [x] I have tested my changes, including edge cases.
- [ ] I have added necessary tests for the changes introduced (if applicable).
- [ ] I have updated the documentation to reflect my changes (if applicable).

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #Tue Jan 02 14:24:44 CET 2024
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
```
Comment (JonatanPlesko) [inline gradle/wrapper/gradle-wrapper.properties]: Why not 8.10?

**Review by JonatanPlesko — APPROVED** [review]

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (KCeh) [reply gradle/wrapper/gradle-wrapper.properties]: I will recheck it, AS AGP upgrade assistant choose it
